app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/commonWidgetControls.jsx

```diff
@@ -132,15 +119,6 @@ export class CommonWidgetControls extends Component {
             <InputTextArea />
           </FieldProvider>
         </ModalField>
-        {this.isShowDashboardsList() && (
-          <FieldProvider
-            name="selectedDashboard"
-            dashboards={dashboards}
-            onChange={onChangeDashboard}
-          >
-            <DashboardControl />
-          </FieldProvider>
-        )}
```

Comment:
Why are you removed this control?
It is needed when user adds widget from launches page.